src/components/app.jsx

```diff
@@ -56,23 +62,37 @@ const PublicContext = withClient(({ client }) => {
   }
 })
 
-const App = ({ isPublic }) => (
-  <HashRouter>
-    <Layout monoColumn={true}>
-      <Main>
-        <Content className="app-content">
-          {isPublic ? <PublicContext /> : <PrivateContext />}
-        </Content>
-      </Main>
-      <IconSprite />
-    </Layout>
-  </HashRouter>
-)
+const App = ({ isPublic, breakpoints: { isMobile }, client }) => {
+  let appName = ''
+  if (isMobile) {
+    const data = client.getInstanceOptions()
+    appName = getDataOrDefault(data.cozyAppName, manifest.name)
```

Comment:
I think we can hardcode the fallback name, no need to load the manifest.

Reply:
This is already done like that in the browser entry point. What is the problem of importing the manifest since it is already imported and parsed

Reply:
More code! But yeah I said this before seeing that the manifest was already imported elsewhere, so not really a problem.